onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -68,7 +68,7 @@ static void RegisterMIGraphXKernels(KernelRegistry& kernel_registry) {
   };
 
   for (auto& function_table_entry : function_table) {
-    kernel_registry.Register(function_table_entry());
+    auto status = kernel_registry.Register(function_table_entry());
```

Comment:
status isn't used?
or do something like
ORT_ENFORCE(kernel_registry.Register(function_table_entry()).IsOK());

Reply:
Thanks. Fixed it and just updated to the latest repo.